src/main/java/seedu/exercise/commons/util/AppUtil.java

```diff
@@ -36,4 +37,13 @@ public static void checkArgument(Boolean condition, String errorMessage) {
             throw new IllegalArgumentException(errorMessage);
         }
     }
+
+    /**
+     * Checks that {@code state} of the program is actually {@code state}. Used for validating a particular
```

Comment:
Weird phrasing here

> Checks that {@code state} of the program is actually {@code state}

Reply:
Also why is ``assert`` used over here?

Reply:
will change the docs there. will change to throw exception to be more consistent with the remaining methods in there. thanks

Reply:
done